content/blog/2020-outreach-partners.md

```diff
@@ -25,6 +23,13 @@ For supporting the conference, our outreach partners can expect following in ret
 * We will display your Logos and Names on the conference website. 
 * We will also promote your community on our social media channels as a supporter of the conference.
 
+## Disclaimer
+ 
+* There will not be any discounts on ticket prices or any other financial benefits of any kind, since PyCon India 2020 is a volunteer driven event.  

```

Comment:
@Sc0rpi0n101 Would it be better to include a link for our deployed about page. That way folks can take reference for what this actually means. Could be "nice to have" but not a blocking change request.

Reply:
But there's already a link to that like 3 lines ahead. Ahould I add it here too?

Reply:
Yes, if it answers their question better then it would be a good idea to provide the information where it's needed more. 
Each one paying for the tickets is a concept that few understand in just one line, we want to be sure to provide enough context to avoid confusion.

Reply:
BTW there's no reference to buying tickets on the about page, It was on the volunteers blog, I think.

Reply:
Gotta start with a FAQ now.

Reply:
Yeah, that's a good idea. We should have an FAQ.